CorsixTH/Lua/entities/object.lua

```diff
@@ -621,6 +622,11 @@ function Object:isReservedFor(user)
   return false
 end
 
+--! Checks the object is a machine.
+function Object:isMachine()
+  return self.strength ~= nil
+end
```

Comment:
Not your fault, but this is weird.

Normally you would make a derived `Machine` class from the `Object` class.
This method then returns `false`, while the same method in `Machine` returns `true`.